fsm/fsm_models.go

```diff
@@ -404,3 +404,13 @@ type FSMSnapshotEvent struct {
 	Input   interface{}
 	Output  interface{}
 }
+
+// StartFSMWorkflowInput should be used to construct the input for any StartWorkflowExecutionRequests.
+// This panics on errors cause really this should never err.
+func StartFSMWorkflowInput(serializer Serialization, data interface{}) *string {
```

Comment:
The serializer being a Serialization here means that you need to pass either an FSM or FSM Context, which should easily be available in most usages.

Could change back to having it be a StateSerializer, and panicing on errors (which is what FSM and context do anyhow)